x/params/store/store.go

```diff
@@ -0,0 +1,167 @@
+package store
+
+import (
+	"fmt"
+	"reflect"
+	"regexp"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/wire"
+)
+
+// Individual parameter store for each keeper
+type Store struct {
```

Comment:
Replace SubStore with Subspace (SubStore is not a types.Store)

Reply:
params.Keeper.SubStore should also be keeper.Subspace.  (no cap on s for space, subspace being a word, and easier to type)